nasa/ow_autonomy
# Oceanwater 724 package rename

This PR is just a renaming and refiling:

- the ow_autonomy package is renamed to ow_plexil
- autonomy_node ROS node is renamed to plexil_node
- a new subdirectory ow_plexil in ow_autonomy now contains all the previous content (

NOTE:  the new subdirectory is the main cause of 78 changed files.  These are mainly moves, with a few files having names and paths changed.

To test:

- catkin clean
- catkin build
- start any simulator world
- start the Plexil node with the following command:  roslaunch ow_plexil plexil_node.launch

If it starts successfully, the test has passed; no need to let it complete.

The user documentation has been updated on a branch of the same name in the ow_simulator.wiki repository.  I don't see a need to review this.   After this PR is merged, I will update any documents on Confluence that use the old names.
